src/app/legacy/components/MediaPlayer/Canonical/index.jsx

```diff
@@ -87,7 +87,10 @@ const Canonical = ({
         allow="autoplay"
         scrolling="no"
         gesture="media"
+        loading="lazy"
         allowFullScreen
+        width="645.25"
+        height="362.953125"
```

Comment:
Are these widths / heights defined to prevent content shift? Presume it shouldn't affect responsiveness having them as fixed values.

Reply:
actually (TIL), lazy loading only works with widths and heights

doesn't affect responsiveness, but it _will_ affect us loading different aspect ratios for the video player, but then there's other things we need to do for that, so cross that bridge when we come to it, I guess...